Makefile

```diff
@@ -0,0 +1,48 @@
+VERSION=$(shell git describe --abbrev=0 --tags)
+BUILD=$(shell git rev-parse HEAD)
+DIRBASE=./build
+DIR=${DIRBASE}/${VERSION}/${BUILD}/bin
+
+LDFLAGS=-ldflags "-s -w ${XBUILD} -buildid=${BUILD} -X github.com/jpillora/chisel/share.BuildVersion=${VERSION}"
+GCFLAGS=-gcflags=all=-trimpath=$(GOPATH)
+ASMFLAGS=-asmflags=all=-trimpath=$(GOPATH)
+
+GOFILES=`go list ./...`
+GOFILESNOTEST=`go list ./... | grep -v test`
+
+# Make Directory to store executables
+$(shell mkdir -p ${DIR})
+
+all: freebsd linux windows
```

Comment:
im okay with having commands for individual OSs

though go-releaser is used to build all https://github.com/jpillora/chisel/blob/master/.github/goreleaser.yml#L1-L3 maybe you could make a `release` command, which executes the `goreleaser.yml` file locally?

Reply:
oh cool - updated!